samples/data-sealing/common/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Use the edger8r to generate C bindings from the EDL file.
+add_custom_command(OUTPUT datasealing_t.h datasealing_t.c
+  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/datasealing.edl)
+
+# Create a library common to each of our three enclaves.
+add_library(common STATIC dispatcher.cpp keys.cpp ${CMAKE_CURRENT_BINARY_DIR}/datasealing_t.c)
+target_compile_definitions(common PUBLIC OE_API_VERSION=2)
+target_link_libraries(common PUBLIC
+  # `liboecore`, a dependency of `liboeenclave`, requires the ecalls
+  # function table. Because the libraries linking `libcommon` do not
+  # directly require this symbol, the linker skips the object in
+  # `libcommon` which defines them. So we manually require it.
+  #
+  # Alternatively we could use a CMake OBJECT library, but that
```

Comment:
3.12 is for the use of `OBJECT` libraries which we didn't require here. I am quite confident this will work with 3.10, and fairly confident it would work down to 3.x ish since it's a standard export. It would be nice to test that, but let's punt it for now?

Reply:
I'm fine with punting, but as always, please track with an issue. We want to get to a point where we can be very crisp on the level of tooling support for SDK package users, and if that's different from maintainers.

Reply:
Got it #1561.